packages/edit-post/src/hooks/commands/use-common-commands.js

```diff
@@ -100,4 +104,22 @@ export default function useCommonCommands() {
 			close();
 		},
 	} );
+
+	useCommand( {
+		name: 'core/edit-post/open-preferences',
+		label: __( 'Editor preferences' ),
```

Comment:
That label is weird, should it be "Toggle editor preferences modal" or "Open editor preferences modal". Also should it be a "toggle" that way we close the modal if it's already opened?

(Same for the other modal)